drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/aha.rb

```diff
@@ -0,0 +1,64 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module HmisExternalApis::AcHmis
+  class Aha
+    SYSTEM_ID = 'ac_hmis_aha'
+
+    Error = HmisErrors::ApiError.new(display_message: 'Failed to connect to AHA')
+
+    def fetch_score(client)
+      mci_ids = client.ac_hmis_mci_ids.map(&:value)
+      return nil if mci_ids.empty?
+
+      payload = {
+        'mc_id__in': mci_ids,
+      }
+
+      result = conn.post('api/v1/clients/scores', payload).
+        then { |r| handle_error(r) }
+
+      return nil if result.http_status == 404
```

Comment:
Add comment- does this mean client was not found? Want to be careful to not swallow server errors. For example in testing I think I was getting 404 resource not found because the API endpoint wasn't up yet. Maybe once it's working we will have a clearer way to differentiate the "no results" response.